plasma_framework/contracts/src/exits/utils/BondSize.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.5.0;
+
+import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+
+library BondSize {
+    using SafeMath for uint256;
+
+    struct Params {
+        uint256 currentGasPrice;
+        uint256 challengeGasCost;
```

Comment:
The naming here can be more generalized and not take in the concept of `challenge` I think (?)